api/ApiRouter.inc

```diff
@@ -73,7 +76,35 @@ class ApiRouter
         throw new InvalidAPI();
     }
 
-    public static function get_router()
+    public function request(bool $raw = false)
```

Comment:
This function could stay in index.php

Reply:
I moved this here to keep the encoding and decoding together. And because I had to move the encoding here this came along with it.

Reply:
That seems reasonable. I've had some further thoughts about this and deleted my other ananswered comments which had overlooked a change you made in index.php. It's difficult to see the whole picture in this github view so I dowloaded the files to look at them more carefully.